odoo/addons/base/migrations/11.0.1.3/post-migration.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import json
+
+from odoo.tools import pickle
+
+from openupgradelib import openupgrade
+
+from psycopg2.extensions import AsIs
+
+
+def map_ir_actions_server_fields(cr):
+    """Map model_id from ir_values to ir_actions' binding_model_id. As per
+    table inheritance, this will set the values in the related tables such
```

Comment:
this is not generally true for Odoo models inheriting other models/tables, but for those specific ones it is, because the init sql does the appropriate trickery: https://github.com/OCA/OCB/blob/11.0/odoo/addons/base/base.sql#L5